create-rust-app/src/dev/mod.rs

```diff
@@ -300,7 +304,7 @@ fn check_exit(state: &DevState) {
 }
 
 async fn listen_for_signals(signal_tx: tokio::sync::broadcast::Sender<DevServerEvent>) {
-    let (ev_s, ev_r) = priority::bounded::<Event, Priority>(1024);
+    let (event_sender, event_receiver) = priority::bounded::<Event, Priority>(1024);
     let (er_s, mut er_r) = tokio::sync::mpsc::channel(64);
```

Comment:
should rename this to `error_sender` and `error_receiver` respectively, again, will do in a different PR

Reply:
fixed in #373